File: JavaToPython/java_to_python11.py
# ChristianThompson.com
# @TokyoEdTech
# Lesson 11: Classes

from animal import Animal

print("Animals (Three Different Ones)")

lucky = Animal("Lucky")
gizmo = Animal("Gizmo")
bella = Animal("Bella")

lucky.set_weight(20.5);
gizmo.set_weight(6.2);
bella.set_weight(5.1);

print("\nNumber of Animals: {}".format(Animal.number_of_animals))

print(lucky)
print(gizmo)
print(bella)

Animal.increment_number_of_animals()
print("\nNumber of Animals: {}".format(Animal.number_of_animals))

File: JavaToPython/JavaToPython11.java
// ChristianThompson.com
// @TokyoEdTech
// Lesson 11: Classes

class Main {
  public static void main(String[] args) {
    System.out.println("Animals (Three Different Ones)\n");

    Animal lucky = new Animal("Lucky");
    Animal gizmo = new Animal("Gizmo");
    Animal bella = new Animal("Bella");

    lucky.setWeight(20.5);
    gizmo.setWeight(6.2);
    bella.setWeight(5.1);

    System.out.println("Number of Animals: " + Animal.numberOfAnimals);
    
    System.out.println(lucky);
    System.out.println(gizmo);
    System.out.println(bella);
    
    Animal.incrementNumberOfAnimals();
    System.out.println("\nNumber of Animals: " + Animal.numberOfAnimals);

  }
}
